Feature/20220524

hyunjunlee123 opened the pull request:
메뉴분리를 위해 서비스게이트웨이 사용자 가이드 다시 작성.

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update console-guide.md

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+## Network > Service Gateway > 콘솔 사용 가이드
+
+본 문서에서는 콘솔에서 Service Gateway를 다룰 때 필요한 내용을 기술합니다.
```
콘솔에서 Service Gateway 서비스를 사용하는 방법을 설명합니다.

youngkyungyu replied:
콘솔에서 Service Gateway 서비스를 사용하는 방법을 설명합니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+## 서비스 게이트웨이
+
+### 서비스 게이트웨이 생성
+아래 설정들을 통하여 **서비스 게이트웨이**를 생성할 수 있습니다.
+* **Network > Service Gateway** 로 이동 합니다.
+* **서비스 게이트웨이 생성** 버튼을 클릭하면 생성 화면이 나타납니다.
+* **서비스 게이트웨이**에 사용할 **이름**을 입력 합니다.
+* **VPC**를 선택합니다.
+  > [참고] 선택된 VPC에 종속된 **서비스 게이트웨이**가 생성됩니다.
+* **서브넷**을 선택합니다.
+  > [참고] 선택된 **서브넷**에서 **서비스 게이트웨이**의 IP가 할당됩니다.
+* **서비스**를 선택합니다.
+  > [참고] **서비스 게이트웨이**에 할당된 IP로 접근시 이 항목에서 선택된 **서비스**와 연결됩니다.
```
서비스 게이트웨이를 생성하는 방법은 다음과 같습니다.
1. **Network > Service Gateway**로 이동합니다.
2. **서비스 게이트웨이 생성** 버튼을 클릭하면 생성 화면이 나타납니다.
3. **서비스 게이트웨이**에 사용할 **이름**을 입력합니다.
4. **VPC**를 선택합니다. 선택된 VPC에 종속된 **서비스     게이트웨이**가 생성됩니다.
5. **서브넷**을 선택합니다. 선택된 **서브넷**에서 **서비스 게이트웨이**의 IP가 할당됩니다.
6. **서비스**를 선택합니다. **서비스 게이트웨이**에 할당된 IP로     접근 시 이 항목에서 선택된 **서비스**와 연결됩니다.

youngkyungyu replied:
서비스 게이트웨이를 생성하는 방법은 다음과 같습니다.
1. **Network > Service Gateway**로 이동합니다.
2. **서비스 게이트웨이 생성** 버튼을 클릭하면 생성 화면이 나타납니다.
3. **서비스 게이트웨이**에 사용할 **이름**을 입력합니다.
4. **VPC**를 선택합니다. 선택된 VPC에 종속된 **서비스     게이트웨이**가 생성됩니다.
5. **서브넷**을 선택합니다. 선택된 **서브넷**에서 **서비스 게이트웨이**의 IP가 할당됩니다.
6. **서비스**를 선택합니다. **서비스 게이트웨이**에 할당된 IP로     접근 시 이 항목에서 선택된 **서비스**와 연결됩니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+  > [참고] **서비스 게이트웨이**에 할당된 IP로 접근시 이 항목에서 선택된 **서비스**와 연결됩니다.
+
+### 서비스 게이트웨이 조회
+생성한 **서비스 게이트웨이**는 **Network > Service Gateway** 페이지에서 확인할 수 있습니다. **서비스 게이트웨이**를 선택하면 하단에 **서비스 게이트웨이** 정보가 나타납니다.
```
페이지에서 확인할 수
=> 화면에서 확인할 수

youngkyungyu replied:
페이지에서 확인할 수
=> 화면에서 확인할 수

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+생성한 **서비스 게이트웨이**는 **Network > Service Gateway** 페이지에서 확인할 수 있습니다. **서비스 게이트웨이**를 선택하면 하단에 **서비스 게이트웨이** 정보가 나타납니다.
+
+### 서비스 게이트웨이 변경
+아래 설정들을 통하여 **서비스 게이트웨이**를 변경할 수 있습니다. **이름**, **설명**만 변경이 가능합니다.
+* **Network > Service Gateway** 로 이동 합니다.
+* **서비스 게이트웨이 변경** 버튼을 클릭하면 변경 화면이 나타납니다.
```
**서비스 게이트웨이**를 변경하는 방법은 다음과 같습니다. **이름**, **설명**만 변경할 수 있습니다.
1. **Network > Service Gateway**로 이동합니다.
2. **서비스 게이트웨이 변경** 버튼을 클릭한 후 변경 화면에서 원하는 항목을 변경합니다.

youngkyungyu replied:
**서비스 게이트웨이**를 변경하는 방법은 다음과 같습니다. **이름**, **설명**만 변경할 수 있습니다.
1. **Network > Service Gateway**로 이동합니다.
2. **서비스 게이트웨이 변경** 버튼을 클릭한 후 변경 화면에서 원하는 항목을 변경합니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+* **서비스 게이트웨이 변경** 버튼을 클릭하면 변경 화면이 나타납니다.
+
+### 서비스 게이트웨이 삭제
+**Network > Service Gateway** 페이지에서 삭제할 **서비스 게이트웨이**를 선택하고 **서비스 게이트웨이 삭제** 버튼을 클릭하면 삭제가 진행됩니다.
```
**서비스 게이트웨이**를 삭제하려면 **Network > Service Gateway** 화면에서 삭제할 **서비스 게이트웨이**를 선택하고 **서비스 게이트웨이 삭제** 버튼을 클릭합니다.

youngkyungyu replied:
**서비스 게이트웨이**를 삭제하려면 **Network > Service Gateway** 화면에서 삭제할 **서비스 게이트웨이**를 선택하고 **서비스 게이트웨이 삭제** 버튼을 클릭합니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+
+## 서비스 게이트웨이 사용
+### 서비스 게이트웨이 IP확인
+* **Network > Service Gateway** 로 이동 합니다.
+* **서비스 게이트웨이** 목록에서 **IP 주소**를 확인합니다.
+  > [참고] 이 **VM Instance**에서 이 **IP주소**로 접속시 **서비스 게이트웨이**가 연결하고 있는 **서비스**로 연결됩니다.
```
1. **Network > Service Gateway**로 이동합니다.
2. **서비스 게이트웨이** 목록에서 **IP 주소**를 확인합니다.
  이 **VM Instance**에서 이 **IP주소**로 접속 시 **서비스 게이트웨이**가 연결하고 있는 **서비스**로 연결됩니다.

youngkyungyu replied:
1. **Network > Service Gateway**로 이동합니다.
2. **서비스 게이트웨이** 목록에서 **IP 주소**를 확인합니다.
  이 **VM Instance**에서 이 **IP주소**로 접속 시 **서비스 게이트웨이**가 연결하고 있는 **서비스**로 연결됩니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+### 서비스 게이트웨이 접속
+생성된 **서비스 게이트웨이**의 **IP주소**가 `192.168.1.42`라 할 경우 다음과 같은 방법으로 서비스에 접근이 가능합니다.
+* **VM Instance**에서 **서비스 게이트웨이 IP**로 접속하면 **서비스 게이트웨이** 생성 시 선택된 **서비스**로 연결되어 **서비스** 사용이 가능합니다.
+    * IP주소를 사용하여 https 프로토콜을 이용할 경우 인증서 관련 에러가 발생할수도 있습니다.
```
* IP 주소를 사용하여 https 프로토콜을 이용할 경우 인증서 관련 오류가 발생할 수도 있습니다.

youngkyungyu replied:
* IP 주소를 사용하여 https 프로토콜을 이용할 경우 인증서 관련 오류가 발생할 수도 있습니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+생성된 **서비스 게이트웨이**의 **IP주소**가 `192.168.1.42`라 할 경우 다음과 같은 방법으로 서비스에 접근이 가능합니다.
+* **VM Instance**에서 **서비스 게이트웨이 IP**로 접속하면 **서비스 게이트웨이** 생성 시 선택된 **서비스**로 연결되어 **서비스** 사용이 가능합니다.
+    * IP주소를 사용하여 https 프로토콜을 이용할 경우 인증서 관련 에러가 발생할수도 있습니다.
+    * https 사용이 필요한 경우 **VM Instance**의 **/etc/hosts**에 IP주소와 URL을 추가하여 사용하시기 바랍니다.
```
* https 사용이 필요한 경우 **VM Instance**의 **/etc/hosts**에 IP 주소와 URL을 추가하시기 바랍니다.

youngkyungyu replied:
* https 사용이 필요한 경우 **VM Instance**의 **/etc/hosts**에 IP 주소와 URL을 추가하시기 바랍니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+* **VM Instance**에서 **서비스 게이트웨이 IP**로 접속하면 **서비스 게이트웨이** 생성 시 선택된 **서비스**로 연결되어 **서비스** 사용이 가능합니다.
+    * IP주소를 사용하여 https 프로토콜을 이용할 경우 인증서 관련 에러가 발생할수도 있습니다.
+    * https 사용이 필요한 경우 **VM Instance**의 **/etc/hosts**에 IP주소와 URL을 추가하여 사용하시기 바랍니다.
+    * 예시) IP주소를 이용하여 **오브젝트 스토리지** 에서 파일 다운로드
```
* 예시) IP 주소를 이용하여 **오브젝트 스토리지**에서 파일 다운로드

youngkyungyu replied:
* 예시) IP 주소를 이용하여 **오브젝트 스토리지**에서 파일 다운로드

youngkyungyu commented on ko/console-guide.md:
````diff
@@ -0,0 +1,101 @@
+        ```
+        ~# wget http://192.168.1.42/v1/AUTH_8222a22c22244badbf876dcd521f3f98/test-obs/test_file.txt
+        ```
+* **서비스 게이트웨이**를 이용하여 **서비스** 접근시 URL을 지원하지 않습니다. URL 접근이 필요한 경우 아래 예시와 같이 /etc/hosts 파일에 URL을 추가해야 합니다.
````
접근시 => 접근 시

youngkyungyu replied:
접근시 => 접근 시

youngkyungyu commented on ko/console-guide.md:
````diff
@@ -0,0 +1,101 @@
+        ~# wget http://192.168.1.42/v1/AUTH_8222a22c22244badbf876dcd521f3f98/test-obs/test_file.txt
+        ```
+* **서비스 게이트웨이**를 이용하여 **서비스** 접근시 URL을 지원하지 않습니다. URL 접근이 필요한 경우 아래 예시와 같이 /etc/hosts 파일에 URL을 추가해야 합니다.
+    * 예시) URL을 이용하여 **오브젝트 스토리지** 에서 파일 다운로드<br>
````
* 예시) URL을 이용하여 **오브젝트 스토리지**에서 파일 다운로드<br>

youngkyungyu replied:
* 예시) URL을 이용하여 **오브젝트 스토리지**에서 파일 다운로드<br>

youngkyungyu commented on ko/console-guide.md:
````diff
@@ -0,0 +1,101 @@
+        ```
+* **서비스 게이트웨이**를 이용하여 **서비스** 접근시 URL을 지원하지 않습니다. URL 접근이 필요한 경우 아래 예시와 같이 /etc/hosts 파일에 URL을 추가해야 합니다.
+    * 예시) URL을 이용하여 **오브젝트 스토리지** 에서 파일 다운로드<br>
+      /etc/hosts 파일에 아래와 같이 **서비스 게이트웨이**의 **IP주소**와 **Object Storage**의 URL을 추가합니다.
````
/etc/hosts 파일에 아래와 같이 **서비스 게이트웨이**의 **IP 주소**와 **Object Storage**의 URL을 추가합니다.

youngkyungyu replied:
/etc/hosts 파일에 아래와 같이 **서비스 게이트웨이**의 **IP 주소**와 **Object Storage**의 URL을 추가합니다.

youngkyungyu commented on ko/console-guide.md:
````diff
@@ -0,0 +1,101 @@
+      ```
+      192.168.1.42    api-storage.cloud.toast.com
+      ```
+      **IP주소** 대신 /etc/hosts에 추가한 URL로 접속
````
**IP 주소** 대신 /etc/hosts에 추가한 URL로 접속

youngkyungyu commented on ko/console-guide.md:
````diff
@@ -0,0 +1,101 @@
+      ```
+      
+## 서비스 게이트웨이에서 오브젝트 스토리지 사용 예제
+오브젝트 스토리지에 관련된 내용은 예제 설명을 위한 수준에서만 기술합니다. 오브젝트 스토리지의 자세한 사용방법은 **사용자 가이드 > Storage > Oject Storage**를 참고하시기 바랍니다.
````
사용방법은 => 사용 방법은

youngkyungyu replied:
사용방법은 => 사용 방법은 => 사용법은

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+      
+## 서비스 게이트웨이에서 오브젝트 스토리지 사용 예제
+오브젝트 스토리지에 관련된 내용은 예제 설명을 위한 수준에서만 기술합니다. 오브젝트 스토리지의 자세한 사용방법은 **사용자 가이드 > Storage > Oject Storage**를 참고하시기 바랍니다.
+### 서비스 게이트웨이 생성 하기
+**오브젝트 스토리지** API를 사용을 위해서는 **인증 토큰**을 발급받아야 합니다. 인터넷 사용이 불가능한 격리된 환경의 **VPC**에서 **Object Storage** 사용을 위해서는 **인증 토큰**도 **서비스 게이트웨이**를 이용하여 발급 받아야하며, 다음 절차에 따라서 **서비스 게이트웨이**를 생성해야합니다.
+* **Object Storage** 서비스를 선택하여 **서비스 게이트웨이** 생성
+  > [참고] **오브젝트 스토리지** API 접속을 위한 **서비스 게이트웨이** 입니다.
+* **IaaS API Identify** 서비스를 선택하여 **서비스 게이트웨이** 생성
+  > [참고] **인증 토큰(token)** 발급을 위한 **서비스 게이트웨이** 입니다.
+* 만들어진 두개의 **서비스 게이트웨이**에서 **IP주소** 확인
```
### 서비스 게이트웨이 생성
**오브젝트 스토리지** API를 사용하려면 **인증 토큰**을 발급받아야 합니다. 인터넷 사용이 불가능한 격리된 환경의 **VPC**에서 **Object Storage**를 사용하려면 **인증 토큰**도 **서비스 게이트웨이**를 이용하여 발급받아야 하며, 다음 절차에 따라서 **서비스 게이트웨이**를 생성해야 합니다.

1. **Object Storage** 서비스를 선택하여 **서비스 게이트웨이**를 생성합니다.
  **오브젝트 스토리지** API 접속을 위한 **서비스 게이트웨이**입니다.
2. **IaaS API Identify** 서비스를 선택하여 **서비스 게이트웨이**를 생성합니다.
  **인증 토큰(token)** 발급을 위한 **서비스 게이트웨이**입니다.
3. 생성된 두 개의 **서비스 게이트웨이**에서 **IP 주소**를 확인합니다.

youngkyungyu replied:
### 서비스 게이트웨이 생성
**오브젝트 스토리지** API를 사용하려면 **인증 토큰**을 발급받아야 합니다. 인터넷 사용이 불가능한 격리된 환경의 **VPC**에서 **Object Storage**를 사용하려면 **인증 토큰**도 **서비스 게이트웨이**를 이용하여 발급받아야 하며, 다음 절차에 따라서 **서비스 게이트웨이**를 생성해야 합니다.

1. **Object Storage** 서비스를 선택하여 **서비스 게이트웨이**를 생성합니다.
  **오브젝트 스토리지** API 접속을 위한 **서비스 게이트웨이**입니다.
2. **IaaS API Identify** 서비스를 선택하여 **서비스 게이트웨이**를 생성합니다.
  **인증 토큰(token)** 발급을 위한 **서비스 게이트웨이**입니다.
3. 생성된 두 개의 **서비스 게이트웨이**에서 **IP 주소**를 확인합니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+* 만들어진 두개의 **서비스 게이트웨이**에서 **IP주소** 확인
+
+### /etc/hosts 파일 편집
+**Object Storage**를 선택하여 생성한 **서비스 게이트웨이**의 IP주소가 192.168.1.42이고 **IaaS API Identify**를 선택하여 생성한 **서비스 게이트웨이**의 IP주소가 192.168.1.57을 할당 받은 경우 **VM Instance**의 /etc/hosts 파일에 아래와 같이 IP주소와 URL을 추가합니다.
```
**Object Storage**를 선택하여 생성한 **서비스 게이트웨이**의 IP 주소가 192.168.1.42이고 **IaaS API Identify**를 선택하여 생성한 **서비스 게이트웨이**의 IP 주소로 192.168.1.57을 할당받은 경우, **VM Instance**의 /etc/hosts 파일에 아래와 같이 IP 주소와 URL을 추가합니다.

youngkyungyu replied:
**Object Storage**를 선택하여 생성한 **서비스 게이트웨이**의 IP 주소가 192.168.1.42이고 **IaaS API Identify**를 선택하여 생성한 **서비스 게이트웨이**의 IP 주소로 192.168.1.57을 할당받은 경우, **VM Instance**의 /etc/hosts 파일에 아래와 같이 IP 주소와 URL을 추가합니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+
+### /etc/hosts 파일 편집
+**Object Storage**를 선택하여 생성한 **서비스 게이트웨이**의 IP주소가 192.168.1.42이고 **IaaS API Identify**를 선택하여 생성한 **서비스 게이트웨이**의 IP주소가 192.168.1.57을 할당 받은 경우 **VM Instance**의 /etc/hosts 파일에 아래와 같이 IP주소와 URL을 추가합니다.
+> [참고] **오브젝트 스토리지**의 API URL주소는 콘솔화면의 **Storage > Object Storage**에서 **API 엔드포인트 설정** 버튼을 클릭하여 확인할 수 있습니다.<br>
```
> [참고] **오브젝트 스토리지**의 API URL 주소는 콘솔 화면의 **Storage > Object Storage**에서 **API 엔드포인트 설정** 버튼을 클릭하여 확인할 수 있습니다.<br>

youngkyungyu replied:
> [참고] **오브젝트 스토리지**의 API URL 주소는 콘솔 화면의 **Storage > Object Storage**에서 **API 엔드포인트 설정** 버튼을 클릭하여 확인할 수 있습니다.<br>

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+### /etc/hosts 파일 편집
+**Object Storage**를 선택하여 생성한 **서비스 게이트웨이**의 IP주소가 192.168.1.42이고 **IaaS API Identify**를 선택하여 생성한 **서비스 게이트웨이**의 IP주소가 192.168.1.57을 할당 받은 경우 **VM Instance**의 /etc/hosts 파일에 아래와 같이 IP주소와 URL을 추가합니다.
+> [참고] **오브젝트 스토리지**의 API URL주소는 콘솔화면의 **Storage > Object Storage**에서 **API 엔드포인트 설정** 버튼을 클릭하여 확인할 수 있습니다.<br>
+> [주의] 리전마다 사용하는 **오브젝트 스토리지 API**의 URL주소는 다르기 때문에 **API 엔드포인트 설정**의 URL을 반드시 확인해주시기 바랍니다.
```
> [주의] 리전마다 사용하는 **오브젝트 스토리지 API**의 URL 주소는 다르기 때문에 **API 엔드포인트 설정**의 URL을 반드시 확인해 주시기 바랍니다.

youngkyungyu replied:
> [주의] 리전마다 사용하는 **오브젝트 스토리지 API**의 URL 주소는 다르기 때문에 **API 엔드포인트 설정**의 URL을 반드시 확인해 주시기 바랍니다.

youngkyungyu commented on ko/console-guide.md:
````diff
@@ -0,0 +1,101 @@
+```
+
+### 인증 토큰 발급
+아래 설정을 통해 **오브젝트 스토리지**의 **API 비밀번호 설정**을 하고 **인증 토큰**을 발급 받는다.
````
**오브젝트 스토리지**의 **API 비밀번호 설정**을 하고 **인증 토큰**을 발급받습니다.

youngkyungyu replied:
**오브젝트 스토리지**의 **API 비밀번호 설정**을 하고 **인증 토큰**을 발급받습니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+
+### 인증 토큰 발급
+아래 설정을 통해 **오브젝트 스토리지**의 **API 비밀번호 설정**을 하고 **인증 토큰**을 발급 받는다.
+* **Storage > Object Storage**에서 **API 엔드포인트 설정** 버튼을 클릭
```
1. **Storage > Object Storage**에서 **API 엔드포인트 설정** 버튼을 클릭합니다.

youngkyungyu replied:
1. **Storage > Object Storage**에서 **API 엔드포인트 설정** 버튼을 클릭합니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+### 인증 토큰 발급
+아래 설정을 통해 **오브젝트 스토리지**의 **API 비밀번호 설정**을 하고 **인증 토큰**을 발급 받는다.
+* **Storage > Object Storage**에서 **API 엔드포인트 설정** 버튼을 클릭
+* **API 엔드포인트 설정** 화면의 **API 비밀번호 설정** 란에 사용할 비밀번호를 입력하고 **변경** 버튼 클릭
```
2. **API 엔드포인트 설정** 화면의 **API 비밀번호 설정**에 사용할 비밀번호를 입력하고 **변경** 버튼을 클릭합니다.

youngkyungyu replied:
2. **API 엔드포인트 설정** 화면의 **API 비밀번호 설정**에 사용할 비밀번호를 입력하고 **변경** 버튼을 클릭합니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+아래 설정을 통해 **오브젝트 스토리지**의 **API 비밀번호 설정**을 하고 **인증 토큰**을 발급 받는다.
+* **Storage > Object Storage**에서 **API 엔드포인트 설정** 버튼을 클릭
+* **API 엔드포인트 설정** 화면의 **API 비밀번호 설정** 란에 사용할 비밀번호를 입력하고 **변경** 버튼 클릭
+* NHN Cloud 계정과 앞서 설정한 **API 비밀번호 설정**의 비밀번호를 이용하여 **IaaS API Identify** 서비스용 **서비스 게이트웨이**에 토큰 발급 요청
```
3. NHN Cloud 계정과 앞서 설정한 **API 비밀번호 설정**의 비밀번호를 이용하여 **IaaS API Identify** 서비스용 **서비스 게이트웨이**에 토큰 발급을 요청합니다.

youngkyungyu replied:
3. NHN Cloud 계정과 앞서 설정한 **API 비밀번호 설정**의 비밀번호를 이용하여 **IaaS API Identify** 서비스용 **서비스 게이트웨이**에 토큰 발급을 요청합니다.

youngkyungyu commented on ko/console-guide.md:
````diff
@@ -0,0 +1,101 @@
+      ~# curl -X POST -H 'Content-Type:application/json' https://api-identity.infrastructure.cloud.toast.com/v2.0/tokens -d '{"auth": {"tenantId": "2fda9d4b88244a0a92ff23841198e2e6", "passwordCredentials": {"username": "example@nhn.com", "password": "example123"}}}'
+      ```
+    * 응답<br>
+      아래 응답에서 `access.token.id`항목의 값이 **인증 토큰**입니다. `access.token.expires`에 기록된 시간 까지 **인증 토큰**이 유효합니다.
````
아래 응답에서 `access.token.id` 항목의 값이 **인증 토큰**입니다. `access.token.expires`에 기록된 시간까지 **인증 토큰**이 유효합니다.

youngkyungyu replied:
아래 응답에서 `access.token.id` 항목의 값이 **인증 토큰**입니다. `access.token.expires`에 기록된 시간까지 **인증 토큰**이 유효합니다.

youngkyungyu commented on ko/console-guide.md:
````diff
@@ -0,0 +1,101 @@
+      ```
+      
+### 오브젝트 스토리지 API 사용
+**인증 토큰** 발급을 마쳤으면 **오브젝트 스토리지** API를 사용할 수 있습니다. **오브젝트 스토리지**에 example 이라는 **컨테이너**를 생성하고 test_file.txt를 넣어 놨다고 가정 할 경우 아래와 같은 API 사용방법으로 **컨테이너**에 들어있는 파일을 조회 할수 있습니다.
````
**인증 토큰** 발급을 마쳤으면 **오브젝트 스토리지** API를 사용할 수 있습니다. **오브젝트 스토리지**에 example이라는 **컨테이너**를 생성하고 test_file.txt를 넣어 놨다고 가정할 경우, 아래와 같은 API 사용법으로 **컨테이너**에 있는 파일을 조회할 수 있습니다.

youngkyungyu replied:
**인증 토큰** 발급을 마쳤으면 **오브젝트 스토리지** API를 사용할 수 있습니다. **오브젝트 스토리지**에 example이라는 **컨테이너**를 생성하고 test_file.txt를 넣어 놨다고 가정할 경우, 아래와 같은 API 사용법으로 **컨테이너**에 있는 파일을 조회할 수 있습니다.

youngkyungyu commented on ko/console-guide.md:
```diff
@@ -0,0 +1,101 @@
+### 오브젝트 스토리지 API 사용
+**인증 토큰** 발급을 마쳤으면 **오브젝트 스토리지** API를 사용할 수 있습니다. **오브젝트 스토리지**에 example 이라는 **컨테이너**를 생성하고 test_file.txt를 넣어 놨다고 가정 할 경우 아래와 같은 API 사용방법으로 **컨테이너**에 들어있는 파일을 조회 할수 있습니다.
+* 요청<br>
+  `X-Auth-Token` 에 **인증 토큰**을 추가하여 요청
```
`X-Auth-Token`에 **인증 토큰**을 추가하여 요청

youngkyungyu replied:
`X-Auth-Token`에 **인증 토큰**을 추가하여 요청

youngkyungyu commented on ko/console-guide.md:
````diff
@@ -0,0 +1,101 @@
+  ~# curl -X GET -H 'X-Auth-Token:gAAAAABiVnmCOJVJhh1W2eXGo3aL0eaZxXmd-SMDMIE3zmip2lXy6eH0BlZAlTZBG20dWEm7TF4zi4YIOTKnc6yKh_wqZsyxgMWKkpVNShzE-k6GaSThBP54QeUePSjC2t-R10X6G4xL_Wecl-V-lV-bnOfVo6Ccpz6rv9eLYJnbJw7KrIMSSiY' https://api-storage.cloud.toast.com/v1/AUTH_2fda9d4b88244a0a92ff23841198e2e6/example
+  ```
+* 응답<br>
+  **오브젝트 스토리지 컨터이너**에 들어있는 파일 목록 확인
````
**오브젝트 스토리지 컨터이너**에 있는 파일 목록 확인

youngkyungyu replied:
**오브젝트 스토리지 컨터이너**에 있는 파일 목록 확인

youngkyungyu commented on ko/overview.md:
```diff
@@ -0,0 +1,18 @@
+## Network > Service Gateway > 개요
+
+**서비스 게이트웨이**는 **플로팅 IP**를 쓰지 않고 트래픽이 인터넷을 경유하지 않고도 **VPC** 외부의 NHN Cloud의 서비스를 이용할 수 있게 합니다. **서비스 게이트웨이 생성** 시 선택된 **서비스**와 자동으로 할당된 IP는 1:1 연결 관계를 유지하며, **VPC**에서는 **서비스 게이트웨이**의 IP를 이용하여 NHN Cloud의 내부 네트워크를 경유해 대상 서비스를 안전하게 이용할 수 있습니다. 
```
**Service Gateway** 서비스를 이용하면 **플로팅 IP**를 쓰지 않고 트래픽이 인터넷을 경유하지 않고도 **VPC** 외부의 NHN Cloud의 서비스를 이용할 수 있습니다. **서비스 게이트웨이 생성** 시 선택된 **서비스**와 자동으로 할당된 IP는 1:1 연결 관계를 유지하며, **VPC**에서는 **서비스 게이트웨이**의 IP를 이용하여 NHN Cloud의 내부 네트워크를 경유해 대상 서비스를 안전하게 이용할 수 있습니다.

youngkyungyu replied:
**Service Gateway** 서비스를 이용하면 **플로팅 IP**를 쓰지 않고 트래픽이 인터넷을 경유하지 않고도 **VPC** 외부의 NHN Cloud의 서비스를 이용할 수 있습니다. **서비스 게이트웨이 생성** 시 선택된 **서비스**와 자동으로 할당된 IP는 1:1 연결 관계를 유지하며, **VPC**에서는 **서비스 게이트웨이**의 IP를 이용하여 NHN Cloud의 내부 네트워크를 경유해 대상 서비스를 안전하게 이용할 수 있습니다.

youngkyungyu commented on ko/overview.md:
```diff
@@ -0,0 +1,18 @@
+### 주요 기능
+* **VPC**에서 인터넷을 경유하지 않고 **서비스 게이트웨이**에서 제공하는 NHN Cloud의 **서비스**에 연결할 수 있습니다.
+* **서비스 게이트웨이 생성** 시 서비스 목록에 제공되는 서비스만 이용이 가능합니다.
+* **서비스 게이트웨이**의 **IP주소**는 서**비스 게이트웨이 생성** 시 선택된 **서비스**와 1:1로 연결됩니다.
```
* **서비스 게이트웨이**의 **IP 주소**는 **서비스 게이트웨이 생성** 시 선택된 **서비스**와 1:1로 연결됩니다.

youngkyungyu replied:
* **서비스 게이트웨이**의 **IP 주소**는 **서비스 게이트웨이 생성** 시 선택된 **서비스**와 1:1로 연결됩니다.

youngkyungyu commented on ko/overview.md:
```diff
@@ -0,0 +1,18 @@
+* **서비스 게이트웨이 생성** 시 서비스 목록에 제공되는 서비스만 이용이 가능합니다.
+* **서비스 게이트웨이**의 **IP주소**는 서**비스 게이트웨이 생성** 시 선택된 **서비스**와 1:1로 연결됩니다.
+* **서비스 게이트웨이**는 **서비스 게이트웨이**가 생성된 **VPC** 내에서만 사용이 가능합니다.
+* **VPC**의 **VM Instance**에서 **서비스 게이트웨이**의 **IP주소**로 통신시 **서비스 게이트웨이**가 연결하고 있는 **서비스**와 통신이 됩니다.
```
* **VPC**의 **VM Instance**에서 **서비스 게이트웨이**의 **IP 주소**로 통신 시 **서비스 게이트웨이**와 연결된 **서비스**와 통신됩니다.

youngkyungyu replied:
* **VPC**의 **VM Instance**에서 **서비스 게이트웨이**의 **IP 주소**로 통신 시 **서비스 게이트웨이**와 연결된 **서비스**와 통신됩니다.

youngkyungyu commented on ko/overview.md:
```diff
@@ -0,0 +1,18 @@
+
+### 제공 서비스
+현재는 **서비스 게이트웨이**를 이용하여 다음과 같은 NHN Cloud의 서비스를 사용할 수 있습니다. 점차 다른 서비스도 지원할 예정입니다.
+서비스 명  | 제공 리전 | 설명
```
서비스 명 => 서비스명

youngkyungyu replied:
서비스 명 => 서비스명

youngkyungyu commented on ko/overview.md:
```diff
@@ -0,0 +1,18 @@
+서비스 명  | 제공 리전 | 설명
+------------- | ------------- | -------------------
+Object Storage|한국(판교), 한국(평촌)|**오브젝트 스토리지**는 NHN Cloud에서 제공하는 많은 양의 데이터를 저장할 수 있는 온라인 저장소 서비스입니다. **VPC**의 **VM Instance**에서 인터넷을 경유하지 않고 **오브젝트 스토리지**에 접근이 필요한 경우 **Object Storage**를 선택하여 **서비스 게이트웨이**를 생성합니다.<br>자세한 사용방법은 **사용자 가이드 > Storage > Object Storage**를 참고 하시기 바랍니다.
+IaaS API Identify|한국(판교), 한국(평촌)|**인즌 토큰** 서비스입니다. **VPC** 내부의 **VM Instance**에서 인터넷을 경유하지 않고 **오브젝트 스토리지 API** 이용에 필요한 **인증 토큰**을 발급 받아야 하는 경우 **IaaS API Identify**를 선택하여 **서비스 게이트웨이**를 생성합니다.<br>자세한 사용방법은 **사용자 가이드 > Storage > Object Storage > API 가이드 > 인증 토큰 발급**를 참고 하시기 바랍니다.
```
**인증 토큰** 서비스입니다. **VPC** 내부의 **VM Instance**에서 인터넷을 경유하지 않고 **오브젝트 스토리지 API** 이용에 필요한 **인증 토큰**을 발급받아야 하는 경우, **IaaS API Identify**를 선택하여 **서비스 게이트웨이**를 생성합니다.<br>자세한 사용 방법은 **사용자 가이드 > Storage > Object Storage > API 가이드 > 인증 토큰 발급**을 참고하시기 바랍니다.

youngkyungyu replied:
(의견: Object Storage 가이드 링크를 추가해 주시면 독자에게 더 도움이 되겠습니다.)

youngkyungyu commented on ko/overview.md:
```diff
@@ -0,0 +1,18 @@
+현재는 **서비스 게이트웨이**를 이용하여 다음과 같은 NHN Cloud의 서비스를 사용할 수 있습니다. 점차 다른 서비스도 지원할 예정입니다.
+서비스 명  | 제공 리전 | 설명
+------------- | ------------- | -------------------
+Object Storage|한국(판교), 한국(평촌)|**오브젝트 스토리지**는 NHN Cloud에서 제공하는 많은 양의 데이터를 저장할 수 있는 온라인 저장소 서비스입니다. **VPC**의 **VM Instance**에서 인터넷을 경유하지 않고 **오브젝트 스토리지**에 접근이 필요한 경우 **Object Storage**를 선택하여 **서비스 게이트웨이**를 생성합니다.<br>자세한 사용방법은 **사용자 가이드 > Storage > Object Storage**를 참고 하시기 바랍니다.
```
자세한 사용 방법은 **사용자 가이드 > Storage > Object Storage**를 참고하시기 바랍니다.

youngkyungyu replied:
(의견: Object Storage 가이드 링크를 추가해 주시면 독자에게 더 도움이 되겠습니다.)

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

hyunjunlee123 committed:
Update console-guide.md

sungchang79 committed:
Fix Markdown syntax issues
- Add a blank line between block-level elements
- Adjust indentation for indented code blocks

hyunjunlee123 committed:
Merge pull request #2 from sungchang79/fix-md-syntax

Fix Markdown syntax issues

hyunjunlee123 committed:
Update overview.md

hyunjunlee123 committed:
Update console-guide.md
